metadata-models/src/main/pegasus/com/linkedin/metadata/relationship/Consumes.pdl

```diff
@@ -0,0 +1,11 @@
+namespace com.linkedin.metadata.relationship
+
+/**
+ * A generic model for the
+ */
+@pairings = [ {
+  "destination" : "com.linkedin.common.urn.DataJobUrn",
+  "source" : "com.linkedin.common.urn.DatasetUrn"
```

Comment:
I was a bit unsure about this. The RFC describes the relationship like this, but personally I'd have the urn types flipped, or rename the Relationship to `ConsumedBy`.

Reply:
Agree- either seems like an improvement to me as well

Reply:
I'll flip the order.